subt/docker/base/Dockerfile

```diff
@@ -40,6 +40,8 @@ RUN /bin/sh -c 'echo "deb [trusted=yes] http://packages.ros.org/ros/ubuntu $(lsb
     python-rosinstall \
     ros-melodic-desktop \
     ros-melodic-joystick-drivers \
+    ros-melodic-octomap-server \
```

Comment:
@zwn can you please then update base image? (we should then re-link robotika image on top of that)